src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -148,6 +157,13 @@ def load_command_table(self, _):
         g.custom_command('result', 'aks_command_result',
                          supports_no_wait=False, table_transformer=aks_run_command_result_format)
 
+    # AKS snapshot commands
+    with self.command_group('aks snapshot', snapshot_sdk, client_factory=cf_snapshots) as g:
+        g.custom_command('list', 'aks_snapshot_list', table_transformer=aks_list_snapshot_table_format)
+        g.custom_show_command('show', 'aks_snapshot_show', table_transformer=aks_show_snapshot_table_format)
+        g.custom_command('create', 'aks_snapshot_create', supports_no_wait=True)
+        g.custom_command('delete', 'aks_snapshot_delete', supports_no_wait=True)
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command

Reply:
Most aks commands use in custom.py uses prompt_y_n to do confirmation. This command follow the existing pattern.

from knack.prompting import NoTTYException, prompt_y_n